revenuecat_examples/purchase_tester/android/app/proguard-rules.pro

```diff
@@ -0,0 +1,27 @@
+# Add project specific ProGuard rules here.
+# You can control the set of applied configuration files using the
+# proguardFiles setting in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# If your project uses WebView with JS, uncomment the following
+# and specify the fully qualified class name to the JavaScript interface
+# class:
+#-keepclassmembers class fqcn.of.javascript.interface.for.webview {
+#   public *;
+#}
+
+# Uncomment this to preserve the line number information for
+# debugging stack traces.
+#-keepattributes SourceFile,LineNumberTable
+
+# If you keep the line number information, uncomment this to
+# hide the original source file name.
+#-renamesourcefileattribute SourceFile
+-keepnames class androidx.navigation.fragment.NavHostFragment
+-keepnames class com.revenuecat.purchases.Offering
+
+
+# See: https://github.com/flutter/flutter/issues/127388
+-dontwarn org.kxml2.io.KXmlParser**,org.kxml2.io.KXmlSerializer**
```

Comment:
Do we need these on magic weather too?

Reply:
Good catch. It isn't needed right now because I kept MagicWeatgher at AGP7, and the issue appears with AGP8, but I've preventively added it.